CMakeLists.txt

```diff
@@ -17,6 +17,7 @@ set(TYPESYSTEM_XML_PATH ${PROJECT_SOURCE_DIR}/descriptors/typesystem)
 set(ANNOTATOR_PATH      ${PROJECT_SOURCE_DIR}/descriptors/annotators)
 set(ENGINE_PATH         ${PROJECT_SOURCE_DIR}/descriptors/analysis_engines)
 set(CMAKE_MODULE_PATH   ${PROJECT_SOURCE_DIR}/cmake)
+set(CMAKE_CXX_FLAGS    "${CMAKE_CXX_FLAGS} ${MY_CMAKE_CXX_FLAGS}")
```

Comment:
we set this in cmake/robosherlock.cmake.in; could you explain a bit ${MY_CMAKE_CXX_FLAGS}? could this be moved to the same file?